contentgrid-spring-data-support/src/test/resources/application.yml

```diff
@@ -0,0 +1,9 @@
+spring:
+  jpa:
+    show-sql: true
+    hibernate:
+      ddl-auto: create-drop
+  content:
```

Comment:
can be replaced with a single `@SpringBootTest(properties = {"spring.....=fs"})`